### PR TITLE
WP#6547

### PR DESCRIPTION
**Pull request by naglis**
Migrates modules:

- [x] _connector_flow_
- [x] _connector_flow_xls_
- [x] _connector_flow_example_product_

As there are no tests, I've only tested it with _connector_flow_example_product_ manually. Haven't used `connector_flow` before, so any hints on possible bugs are very appreciated.

### CODE REVIEW

File: connector_flow/models/impexp_task.py [hunk 1]
```diff
@@ -80,40 +68,41 @@ def _get_available_tasks(self):
...
            flow in a task flow"""
-        if self.flow_start:
-            flow_start_count = self.search_count(
-                [('flow_id', '=', self.flow_id.id),
-                 ('flow_start', '=', True)])
-            if 1 < flow_start_count:
-                raise exceptions. \
-                    ValidationError(_('The start of a task flow '
-                                      'has to be unique'))
+        for rec in self:
```
Comment (taezaz) [inline connector_flow/models/impexp_task.py]: self.filtered('flow_start')

File: connector_flow/task/csv_import.py [hunk 2]
```diff
@@ -18,13 +18,15 @@
 #
 ##############################################################################
 
-from openerp import models, api
-from .abstract_task import AbstractTask
-
-from base64 import b64decode
+import base64
```
Comment (taezaz) [inline connector_flow/task/csv_import.py:21]: Why not use 'from' here? It's fine with me, just interested in the reason.

**Review by taezaz — APPROVED** [review]

File: connector_flow/task/csv_import.py (hunk 2 above)
Comment (naglis) [reply connector_flow/task/csv_import.py:21]: Personal preference.

**Review by kkarolis — APPROVED** [review]
Good job.

**Pull request merged**